tensorflow_hub/resolver.py

```diff
@@ -502,6 +504,7 @@ class HttpResolverBase(Resolver):
 
   def __init__(self):
     self._context = None
+    self._maybe_disable_cert_validation()
```

Comment:
Can't we remove this line since the context is maybe already disabled in `_set_url_context`?